test/integration/package.json

```diff
@@ -16,7 +16,7 @@
   "devDependencies": {
     "@apollo/client": "^3.10.4",
     "@interledger/http-signature-utils": "2.0.2",
-    "@interledger/open-payments": "^6.11.1",
+    "@interledger/open-payments": "6.11.1",
```

Comment:
As I recall I did this in the middle of fixing the open payments issue we saw related to the ESM import. I noticed the others were pinned as well so I changed this one but at this point it may not matter.

Reply:
I see, they were always pinned. I guess it's ok.